module-types/Cisco/N7K-SUP2.yaml

```diff
@@ -0,0 +1,12 @@
+---
+manufacturer: Cisco
+model: N7K-SUP2
+part_number: N7K-SUP2
+console-ports:
+  - name: console{module}
+    type: rj-45
+  - name: com1{module}
+    type: rj-45
+interfaces:
+  - name: mgmt0
```

Comment:
This will result in a name collision, please change this to somehow utilize the {module} to prevent this.